storage/innobase/include/fsp0fsp.h

```diff
@@ -755,6 +755,20 @@ fsp_flags_match(ulint expected, ulint actual)
 	return(actual == expected);
 }
 
+/** Determine if FSP_SPACE_FLAGS are from an incompatible MySQL format.
+@param[in]	flags	the contents of FSP_SPACE_FLAGS
+@return	MySQL flags shifted.
+@retval	0, if not a MySQL incompatible format. >0 */
+MY_ATTRIBUTE((warn_unused_result, const))
+inline ulint fsp_flags_is_incompatible_mysql(ulint flags)
+{
+  /*
+    MySQL-8.0 SDI flag (bit 14),
+    or MySQL 5.7 Encyption flag (bit 13)
+  */
+  return (flags >> 13) & 0b11;
```

Comment:
`0b11` needs to be `3` in 10.3, to avoid a dependency on C++11. Parentheses before the `&` are redundant.